openedx/features/course_experience/tests/views/test_course_outline.py

```diff
@@ -200,9 +200,8 @@ def test_num_graded_problems(self):
     @ddt.data(
         ([CourseMode.AUDIT, CourseMode.VERIFIED], CourseMode.AUDIT, False, True),
         ([CourseMode.AUDIT, CourseMode.VERIFIED], CourseMode.VERIFIED, False, True),
-        ([CourseMode.AUDIT, CourseMode.VERIFIED, CourseMode.MASTERS], CourseMode.MASTERS, False, True),
```

Comment:
Masters only is a thing so seemed easier to just simplify this line and make it more readable.